pkg/search/query_builder.go

```diff
@@ -263,6 +263,14 @@ func (qb *QueryBuilder) AddExactMatches(k FieldLabel, values ...string) *QueryBu
 	return qb
 }
 
+// AddExactMatchesUUID adds a key value pair to the query
+func (qb *QueryBuilder) AddExactMatchesUUID(k FieldLabel, values ...string) *QueryBuilder {
```

Comment:
If there is no different between this and the AddExactMatches, you can just call it internally, just to have a name difference.

Reply:
That is a left over artifact from earlier work.  It wasn't being used at all.  So removed.